charts/tempo-distributed/templates/metamonitoring/kube-state-metrics-servmon.yaml

```
@@ -0,0 +1,44 @@
+{{- if and ((.Values.metaMonitoring).grafanaAgent).enabled ((((.Values.metaMonitoring).grafanaAgent).metrics).scrapeK8s).enabled }}
```

whats the motivation for all the extra `()` here? Compared to:
```suggestion
{{- if and ((.Values.metaMonitoring).grafanaAgent).enabled .Values.metaMonitoring.grafanaAgent.metrics.scrapeK8s.enabled }}
```